### ISSUE TITLE
Any chance to add GetPowerFlowRealtimeData

### ISSUE THREAD

**Issue by Fungusware** [SeanoNET/FroniusSolarClient#1]
GetPowerFlowRealtimeData would be a nice addition to the API

**Comment by MowedMeadow** [comment]
I also was searching for GetPowerFlowRealtimeData(). Probably would take a long time for me to develope. Is there something in pipeline?
Anyway, thanks for that great api!

**Comment by Fungusware** [comment]
I've been using this code, it quite quick and dirty but might help you out.

```
var jsonData = client.GetStringAsync($"http://<<HOST:PORT>>/solar_api/v1/GetPowerFlowRealtimeData.fcgi").Result;

var response = Newtonsoft.Json.JsonConvert.DeserializeObject<dynamic>(jsonData);

var site = response.Body.Data.Site;

var timeStampString = (string)response.Head.Timestamp;
var timeStamp = DateTime.ParseExact(timeStampString, "MM/dd/yyyy HH:mm:ss", null);

var p_Grid = site.P_Grid.Value == null ? 0 : (float)site.P_Grid.Value;
var p_Load = site.P_Load.Value == null ? 0 : (float)site.P_Load.Value;
var p_PV = site.P_PV.Value == null ? 0 : (float)site.P_PV.Value;

var autonomy = site.rel_Autonomy.Value == null ? 0 : (float)site.rel_Autonomy.Value;
var selfConsumption = site.rel_SelfConsumption.Value == null ? 0 : (float)site.rel_SelfConsumption.Value;
```

**Comment by MowedMeadow** [comment]
Works for me, thank's for that quick help!

**Comment by SeanoNET** [comment]
Hey @Fungusware and @MowedMeadow 

Sorry, i didn't get a notification for this issue for some reason. I will look at adding support for this method when i get some free time.

**Comment by SeanoNET** [comment]
@Fungusware this has been released in [0.3.0](https://github.com/SeanoNET/FroniusSolarClient/releases/tag/0.3.0)
